src/views/SettingsPage.vue

```diff
@@ -24,8 +24,7 @@
             label="Color Theme"
             aria-label="theme"
             interface="popover"
-            :value="selectValue"
-            @ionChange="selectChanges($event)"
+            v-model="themeType"
```

Comment:
is this `themeType` global accessible as `STORAGE_KEYS.THEME_TYPE)`？Or is this just a local variable from line 73?

Reply:
Yes, it is just the local ref from line 73. I just used Vue's two-way binding instead of using one-way binding and an event of updating the value.